.github/workflows/build_and_release.yml

```
@@ -50,11 +67,41 @@ jobs:
           rm target/${{ matrix.job.target }}/release/litra.d
           cp target/${{ matrix.job.target }}/release/litra* litra_${{ steps.sanitise_ref.outputs.value }}_${{ matrix.job.binary_name }}
         if: runner.os != 'Windows'
+      - name: Write Apple signing key to a file (macOS only)
+        env:
+          APPLE_SIGNING_KEY_P12: ${{ secrets.APPLE_SIGNING_KEY_P12 }}
+        run: echo $APPLE_SIGNING_KEY_P12 | base64 -d -o key.p12
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting

(shellcheck)